wies/core/models.py

```diff
@@ -63,7 +63,11 @@ class Colleague(models.Model):
     # placements via reversed relation
 
     def get_absolute_url(self):
-        return reverse("colleague-detail", kwargs={"pk": self.pk})
+        # Find a placement for this colleague to use in the sidepanel URL
```

Comment:
this comment shows that this is functionality specific for the sidepanel. by implementing it on the model, anyone that needs a URL for this resource gets this url. this logic is too specific for the placements page to be inserted here. you can add a get_colleague_url method in the placementlist view, similarly to the close url?